### DISCUSSION TITLE
Remove Header Link

### DISCUSSION THREAD

**Discussion by ballarddawn** [code4recovery/12-step-meeting-list#742] [Q&A] [answered]
I am a self-teaching AA website developer and have been working with this app - love it! I can't seem to remove the link from the header of the page. I've looked everywhere! Site is https://aaseiowa.com/meetings/. I want the top header on every page but want to remove the link just below. Thank you!!!!!!! 
![2022-03-18 11_12_14-Meetings - AA SE Iowa](https://user-images.githubusercontent.com/101887014/159040721-035aa56d-c1b9-427b-bc5f-3d7dc643e013.png)

**Comment by tech2serve** [comment]
Hi @ballarddawn, I just looked at your site, and I don't see the extra link. I'm assuming you got it figured out?

Perhaps come back here and post what you discovered as a new answer so others might learn from your work?

However, if you're still seeing the extra link when you view the page, please clear your browser cache and see if it clears up. 

Still a problem? Let me know and include what browser you're using, the theme, and version numbers, and we'll have to explore more.

**Reply by ballarddawn** [reply]
A little bird helped me out with some CSS code. :)

**Reply by kiyote33** [reply]
I'm guessing something like #entry-header { display:none; } in the Customizer Additional CSS feature...

**Reply by brianw-area24** [reply]
Glad this is working for you! However, when I look at the page source, I'm seeing what looks like a page embedded within a page. (I'm seeing 2 <html> tags, 2 <body> tags, etc.)Not sure how that is happening. If this page causes problems for you down the road, we may need to look into it a little farther...

**Comment by joshreisner** [comment, marked as answer]
i helped her over here in this thread on TiAA: https://discourse.tiaa-forum.org/t/beginner-help-needed/7365/6
